File: Python/stack_using_array.py
class Stack:
    def __init__(self):
        self.stack = list()
        
    def insertElement(self, data):
        self.stack.append(data)
    
    def deleteElement(self):
        print("Deleted element : ",self.stack.pop(), "\n\n")
    
    def printStack(self):
        print("Stack has elements : ", end=" ")
        for ele in self.stack:
            print(ele, end=", ")
        print("\n")
        
        
if __name__ == "__main__":
    lst = Stack()
    
    while True:
        print("Select the any option")
        print("[1] Insert Element")
        print("[2] Delete Element")
        print("[3] Print List")
        print("[4] Exit")
        print("\n")
        i = int(input())
        print("\n")

        if i == 1:
            lst.insertElement(input("Enter data you want to insert: "))
        elif i == 2:
            lst.deleteElement()
        elif i == 3:
            lst.printStack()
        elif i == 4:
            break
        

File: JavaScript/stack_using_array.js
class Stack {
  constructor() {
    this.stack = [];
  }

  isEmpty() {
    return this.stack.length === 0;
  }

  push(data) {
    this.stack.push(data);
  }

  pop() {
    if (this.stack.length === 0) {
      return "Stack is underflow i.e its empty";
    }
    return this.stack.pop();
  }

  print() {
    if (this.stack.length === 0) {
      console.log("Stack is underflow i.e its empty");
    } else {
      console.log(this.stack);
    }
  }
}

let stack = new Stack();
